### PR TITLE
[WIP] Add and remove agenda items

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Add agenda_item: Should fix [OpenSlides-action-crew/openslides-backend-coordination#75](https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/75)

### CODE REVIEW

File: client/src/app/core/repositories/agenda/agenda-item-repository.service.ts [hunk 1]
```diff
@@ -128,13 +128,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
     }
 
     public async addItemToAgenda(contentObject: BaseViewModel & HasAgendaItem): Promise<Identifiable> {
-        return await this.create({ id: contentObject.id } as AgendaItem);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_CREATE, {
+            content_object_id: `${contentObject.collection}/${contentObject.id}`,
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/agenda/agenda-item-repository.service.ts:132]: Please have a look at `key-transforms.ts` and (maybe) add a method `fqidFromCollectionAndId`.

File: client/src/app/core/repositories/agenda/agenda-item-repository.service.ts [hunk 2]
```diff
@@ -128,13 +128,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
...
+            meeting_id: this.activeMeetingService.meetingId
+        });
     }
 
     public async removeFromAgenda(item: ViewAgendaItem): Promise<void> {
-        return await this.delete(item);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_DELETE, {
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/agenda/agenda-item-repository.service.ts:138]: A note for testing: deleting an agenda item can have two different effects: Deleting a topic must delete the topic, too. Deleting models of other collections just the ageda item has to be deleted.

File: client/src/app/core/repositories/agenda/agenda-item-repository.service.ts [hunk 3]
```diff
@@ -128,13 +128,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
     }
 
     public async addItemToAgenda(contentObject: BaseViewModel & HasAgendaItem): Promise<Identifiable> {
-        return await this.create({ id: contentObject.id } as AgendaItem);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_CREATE, {
+            content_object_id: fqidFromCollectionAndId(contentObject.collection, contentObject.id),
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/agenda/agenda-item-repository.service.ts:132]: This should work: `contentObject.fqid`. Plase have a look in `base-model.ts` and use your new function there (In the file it is still hard-coded).

**Comment by GabrielInTheWorld** [comment]
Duplicate of #41

**Pull request closed without merging**